lib/node/rpc.js

```diff
@@ -2196,6 +2197,39 @@ class RPC extends RPCBase {
     return false;
   }
 
+  async verifyMessageWithName(args, help) {
+    if (help || args.length !== 3) {
+      throw new RPCError(errs.MISC_ERROR,
+        'verifymessagewithname "name" "signature" "message"');
+    }
+
+    const valid = new Validator(args);
+    const name = valid.str(0, '');
+    const sig = valid.buf(1, null, 'base64');
+    const str = valid.str(2);
+
+    if (!name || !rules.verifyName(name))
+      throw new RPCError(errs.TYPE_ERROR, 'Invalid name.');
+
+    const nameHash = rules.hashName(name);
+    const ns = await this.chain.db.getNameState(nameHash);
+
+    if (!ns || !ns.owner)
+      throw new RPCError(errs.MISC_ERROR, 'Cannot find the name owner.');
+
+    const coin = await this.chain.getCoin(ns.owner.hash, ns.owner.index);
+
+    if (!coin) {
+      throw new RPCError(
+        errs.DATABASE_ERROR,
+        'Cannot find the owner\'s address.'
+      );
+    }
+
+    const address = coin.address.toString(this.network);
+    return this.verifyMessage([address, sig, str], help);
```

Comment:
I don't think you need to pass `help` here. If help were `true`, we'd throw an error right away, and we don't want to return help from a different RPC call anyway. Applies to both functions.